src/App.tsx

```diff
@@ -25,38 +26,52 @@ loadUser(store, userManager).then(async user => {
   }
 });
 
+const instance = createInstance({
+  urlBase: 'https://analytics.hel.ninja/',
+  siteId: 59,
+});
+
+// Prevent non-production data from being submitted to Matomo
+// by pretending to require consent to process analytics data and never ask for it.
+// https://developer.matomo.org/guides/tracking-javascript-guide#step-1-require-consent
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
With this solution you'll be sending data from the review & staging environments to matomo. That's ok until we go into production.. especially when you want to test and verify that it works. 

I would recommend adding a new REACT_APP environment variable, for example REACT_APP_ENVIRONMENT where you only submit data for production.

Reply:
Good points! I added event tracking to registration as well. I also added REACT_APP environment variable.